scripts/install-systemd-multi-user.sh

```diff
@@ -47,6 +70,13 @@ poly_configure_nix_daemon_service() {
     _sudo "to set up the nix-daemon socket service" \
           systemctl enable "/nix/var/nix/profiles/default$SOCKET_SRC"
 
+    # Create a systemd unit override with proxy environment variables
```

Comment:
I think this would be nicer in its own function, which is called from here. This way the logic is more tightly captured and makes this function easier to understand.